frontend/src/components/cluster/Overview.tsx

```diff
@@ -125,7 +131,7 @@ function EventsSection() {
           titleSideActions={[
             <FormControlLabel
               checked={isWarningEventSwitchChecked}
-              label={t('Warnings')}
+              label={t('Only warnings ({{ numWarnings }})', { numWarnings })}
```

Comment:
@joaquimrocha we have a filter also to the table where we show events, but this count won't change in case I apply a filter. Should we change that, or should we change this to say total warnings??